### PR TITLE
Add net.pistegamez.PekkaKana2

### PR DESCRIPTION
**Pull request by AsciiWolf**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

This is a game from 2003. Website is still up, however upstream development seems to be dead. There are some forks and patches (for example the one adding SDL2 support), but those are not official and the [only active fork](https://github.com/danilolc/pk2) is completely broken when used in Flatpak. I have based my Flatpak on the [Debian package version](https://packages.debian.org/sid/pekka-kana-2) of Pekka Kana 2, however I was not able to find the original package source.

### CODE REVIEW

File: net.pistegamez.PekkaKana2.desktop [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Pekka Kana 2
+Comment=Jump 'n run game made in the spirit of old classic platformers
```
Comment (AsciiWolf) [inline net.pistegamez.PekkaKana2.desktop:3]: I am not entirely sure whether the apostrophe character is valid in the Comment section of a XDG desktop file. However it does not seem to cause any harm, at least on my (GNOME-based) system.

**Comment by AsciiWolf** [comment]
bot, build net.pistegamez.PekkaKana2

**Comment by AsciiWolf** [comment]
Tested and works fine. :-)

**Comment by orowith2os** [comment]
Isn't working fine over here on GNOME 43, client-side decorations are missing.

Does this use SDL2? If so, please add the SDL2-with-libdecor shared module: https://github.com/flathub/shared-modules/blob/master/SDL2/SDL2-2.26.1-with-libdecor.json

**Comment by AsciiWolf** [comment]
It works fine on my GNOME Wayland session including window (Adwaita) decorations.

**Pull request closed without merging**